indexing-hadoop/src/test/resources/druid.test.data.tsv

```diff
@@ -0,0 +1,82 @@
+2011-04-10T00:00:00.000Z	spot	automotive	preferred	apreferred	113.221448
```

Comment:
Yes:

```
        <dependency>
            <groupId>io.druid</groupId>
            <artifactId>druid-processing</artifactId>
            <version>${project.parent.version}</version>
            <type>test-jar</type>
            <scope>test</scope>
        </dependency>
```

Then `final URL url = Preconditions.checkNotNull(Query.class.getClassLoader().getResource("druid.sample.tsv"));` gives you URL you can open a stream to and copy the data into a temporary folder for unit test processing.